config/_default/config.toml

```diff
@@ -105,33 +105,39 @@ hasChildren = true
 
     [[languages.en.menu.main]]
     parent = "Documentation"
-    name = "v3.1.x <img src='/images/header/star.svg' alt='star'>"
+    name = "v3.2.x <img src='/images/header/star.svg' alt='star'>"
```

Comment:
Can we hide `v3.2.x` for now because it is not ready for release yet? After GA, we can show the  `v3.2.x` menu item.
Same for the ZH website.